bin/github-backup

```
@@ -382,24 +389,67 @@ def get_github_repo_url(args, repository):
     return repo_url
 
 
+def _update_githubapp_token(current_token):
+    global _token_cache
+    token_path = os.path.expanduser('~/.gh-token')
+
+    if os.path.exists(token_path):
+        last_modified = os.path.getmtime(token_path)
+
+        if last_modified > _token_cache['last_modified']:
+            with open(token_path, "r") as f:
+                new_token = f.read().strip()
+            _token_cache['token'] = new_token
+            _token_cache['last_modified'] = last_modified
+
+        if current_token != _token_cache['token']:
+            return _token_cache['token']
+    else:
+        log_error('No token found in ~/.gh-token')
```

I think it should raise an exception or exit in this case. The application can't work without a github token.

---

`def log_error(message):
    if type(message) == str:
        message = [message]

    for msg in message:
        sys.stderr.write("{0}\n".format(msg))

    sys.exit(1)`
    
  log_error() is already doing a sys.exit(1)